administrator/language/de-DE/plg_fields_note.ini

```
@@ -0,0 +1,17 @@
+; Joomla! German Translation
+; (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2025 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_FIELDS_NOTE="Felder – Hinweis"
+PLG_FIELDS_NOTE_LABEL="Hinweis (%s)"
+PLG_FIELDS_NOTE_PARAMS_CLASS_DESC="Die Klasse, die für das div des Hinweises verwendet wird."
+PLG_FIELDS_NOTE_PARAMS_CLASS_LABEL="Hinweis-Klasse"
```

```suggestion
PLG_FIELDS_NOTE_PARAMS_CLASS_LABEL="CSS-Klasse Hinweis"
```
vgl https://github.com/joomlagerman/joomla/blob/8b3a255649b3624caa6d035996faa1428b9a01a4/administrator/language/de-DE/com_menus.ini#L99